src/OpenSearch.Client/Descriptors.Cluster.cs

```diff
@@ -89,7 +89,11 @@ public partial class ClusterGetSettingsDescriptor : RequestDescriptorBase<Cluste
 		///<summary>Whether to return all default clusters setting.</summary>
 		public ClusterGetSettingsDescriptor IncludeDefaults(bool? includedefaults = true) => Qs("include_defaults", includedefaults);
 		///<summary>Explicit operation timeout for connection to master node</summary>
+		///<remarks>Deprecated as of OpenSearch 2.0, use <see cref="ClusterManagerTimeout"/> instead</remarks>
 		public ClusterGetSettingsDescriptor MasterTimeout(Time mastertimeout) => Qs("master_timeout", mastertimeout);
+		///<summary>Explicit operation timeout for connection to cluster_manager node</summary>
+		///<remarks>Introduced in OpenSearch 2.0 instead of <see cref="MasterTimeout"/></remarks>
+		public ClusterGetSettingsDescriptor ClusterManagerTimeout(Time timeout) => Qs("cluster_manager_timeout", timeout);
```

Comment:
Should `MasterTimeout` call `ClusterManagerTimeout` and be deprecated with `@Obsolete`?

Reply:
I didn't use here `Obsolete` attribute, because this is not obsolete for OpenSearch 1.x. This could fail build for users.

I don't call `MasterTimeout` in `ClusterManagerTimeout`, because these methods set different parameters. Imaging you are working with OpenSearch 1.2 and setting `MasterTimeout`. If it sets `cluster_manager_timeout` under the hood`, server won't recognize it.
We can call MasterTimeout` in `ClusterManagerTimeout` *in addition* to setting `cluster_manager_timeout`, but I think it is add.

This is applicable for all instances of `ClusterManagerTimeout` (a lots of them).

Reply:
Ok, you're absolutely right.